examples/scatter-matrix-app.py

```diff
@@ -0,0 +1,42 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/diabetes.csv"
+df = pd.read_csv(filepath)
+# fig = px.scatter_matrix(df)
+# fig.show()
+dimensions_list = df.columns.values
+
+app = Dash(__name__)
+
+app.layout = html.Div([
+    html.H1("Scatterplot Matrix of the Correlates of Diabetes",
+            style={'textAlign': 'center'}),
+    html.Div(
+        [html.H3('Choose your dimensions:',
+                 style={"paddingLight": "60px"}),
+         dcc.Dropdown(
+            id='scatter-matrix-app-x-dimension-dropdown',
+            options=dimensions_list,
+            value=["Outcome", "Glucose", "BMI"],
+            multi=True
+        )], style={"display": "inline-block", "verticalAlign": "top", "width": "30%", "margin-left": "5em"}),
+    dcc.Graph(
+        id='scatter-matrix-app-x-graph'
+    )
+])
+
+
+@ app.callback(
+    Output('scatter-matrix-app-x-graph', 'figure'),
+    [Input("scatter-matrix-app-x-dimension-dropdown", "value")])
+def update_graph(dimension_choice):
+    fig = px.scatter_matrix(df,
+                            dimensions=dimension_choice)
+    fig.update_layout()
```

Comment:
This line has no effect and can be removed.  However when you add lot of options from the dropdown, the y axis labels get hard to read.  One solution might be to increase the height:
```
fig.update_layout(height=600)
```
But I'm sure there are other better options for formatting the y axis labels if you would like to suggestion something else.

Reply:
I just found a way to dynamically adapt the height according to the number of parameters included in the visualization, with height=len(dimension_choice)*166.667, where 166.67 is the quotient of height=500/(initial number of parameters). Let me know what you think and thanks for all your feedback.

Reply:
That's clever! :tada: